generator/model_test.go

```diff
@@ -157,7 +157,7 @@ func TestGenerateModel_SchemaField(t *testing.T) {
 	var gmp GenSchema
 	gmp.Name = "some name"
 	gmp.OriginalName = "some name"
-	gmp.resolvedType = resolvedType{GoType: "string", IsPrimitive: true}
+	gmp.resolvedType = resolvedType{GoType: "string", IsPrimitive: true, IsEmptyOmitted: true}
```

Comment:
is this the new default?

Reply:
yep. By default, x-omitempty is false for array type, true for other.

Reply:
I am puzzled that this does not affect validation code... isn't it strange?

Reply:
The zero value of bool is false. However, our default value is true. 😅

Reply:
no I meant, for null values in arrays. this omitempy stuff is supposed to alter validation right?